scrapi/harvesters/addis_ababa.py

```diff
@@ -6,6 +6,35 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def oai_process_uris_addis_ababa(*args):
+
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    potential_uris = (provider_uris + object_uris)
+
+    for i, uri in enumerate(potential_uris):
+        if 'http://hdl.handle.net/123456789/' in uri:
+            doc_id = potential_uris[i].replace('http://hdl.handle.net/123456789/', '')
```

Comment:
You could also just do;
```python
potential_uris[i].replace('http://handle.net/123456789/', 'http://etd.aa.edu.et/handle/123456789')
```
and drop the string concatenation, since all you are really doing with that is replacing the beginning of the string.